packages/web/src/pages/audio-rewards-page/components/WalletManagementTile.tsx

```diff
@@ -0,0 +1,242 @@
+  advancedOptions: 'Advanced Options',
+  buyWith: 'Buy with',
+  buyWithCoinbase: 'Buy with Coinbase'
+}
+const IS_NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
+
+const AdvancedWalletActions = () => {
+  const balance = useSelector(getAccountBalance) ?? (new BN(0) as BNWei)
+  const hasBalance = balance && !balance.isZero()
+  const dispatch = useDispatch()
+  const [, openTransferDrawer] = useModalState('TransferAudioMobileWarning')
+
+  const mobile = isMobile()
+  const onClickReceive = useCallback(() => {
+    if (mobile) {
+      openTransferDrawer(true)
+    } else {
+      dispatch(pressReceive())
+    }
+  }, [dispatch, mobile, openTransferDrawer])
+
+  const onClickSend = useCallback(() => {
+    if (mobile) {
+      openTransferDrawer(true)
+    } else {
+      dispatch(pressSend())
+    }
+  }, [mobile, dispatch, openTransferDrawer])
+  const [, setOpen] = useModalState('MobileConnectWalletsDrawer')
+
+  const onClickConnectWallets = useCallback(() => {
+    if (mobile) {
+      setOpen(true)
+    } else {
+      dispatch(pressConnectWallets())
+    }
+  }, [mobile, setOpen, dispatch])
+
+  const onCloseConnectWalletsDrawer = useCallback(() => {
+    setOpen(false)
+  }, [setOpen])
+
+  const hasMultipleWallets = useSelector(getHasAssociatedWallets)
+  return (
+    <div className={styles.advancedSection}>
+      <div className={styles.subtitle}>{messages.advancedOptions}</div>
+      <div className={styles.advancedOptions}>
+        <Button
+          className={cn(styles.advancedButton, {
+            [styles.disabled]: !hasBalance
+          })}
+          text={messages.sendLabel}
+          isDisabled={!hasBalance}
+          includeHoverAnimations={hasBalance}
+          textClassName={styles.textClassName}
+          onClick={onClickSend}
+          leftIcon={<IconSend className={styles.iconStyle} />}
+          type={ButtonType.GLASS}
+          minWidth={200}
+        />
+        <Button
+          className={cn(styles.advancedButton, styles.receiveBtn)}
+          text={messages.receiveLabel}
+          textClassName={styles.textClassName}
+          onClick={onClickReceive}
+          leftIcon={<IconReceive className={styles.iconStyle} />}
+          type={ButtonType.GLASS}
+          minWidth={200}
+        />
+        <Button
+          className={cn(styles.advancedButton, styles.connectWalletsBtn)}
+          text={
+            hasMultipleWallets
+              ? messages.manageWallets
+              : messages.connectWallets
+          }
+          includeHoverAnimations
+          textClassName={styles.textClassName}
+          onClick={onClickConnectWallets}
+          type={ButtonType.GLASS}
+          leftIcon={<IconSettings className={styles.iconStyle} />}
+          minWidth={200}
+        />
+        {mobile && !IS_NATIVE_MOBILE && (
+          <MobileConnectWalletsDrawer onClose={onCloseConnectWalletsDrawer} />
+        )}
+      </div>
+    </div>
+  )
+}
+
+const ToggleCollapseButton = ({
```

Comment:
question for julian - do we care about animating this?

Reply:
will check with Julian in a design review: https://linear.app/audius/issue/PAY-551/design-review-for-dollaraudio-page-changes